index.d.ts

```diff
@@ -57,7 +62,7 @@ declare namespace unistUtilIs {
  */
 declare function unistUtilIs<T extends Node>(
   node: unknown,
-  test: unistUtilIs.Test<T> | Array<unistUtilIs.Test<any>>,
+  test?: unistUtilIs.Test<T> | Array<unistUtilIs.Test<any>>,
```

Comment:
Should be `test?: unistUtilIs.Test<T> | Array<unistUtilIs.Test<any>> | null`. And don't forget to fix `convert` function too.

Reply:
This is needed only if `null` and `undefined` are removed from `Test` correct?

Reply:
Yes.